pkg/storaged/jobs.js

```diff
@@ -175,12 +175,19 @@ define([
                 return job(a).StartTime - job(b).StartTime;
             }
 
+            function filter_job(path) {
```

Comment:
Since this isn't a generic filter function, it should have a specific name as well. Maybe something like `filter_short_jobs`? It could also take a threshold parameter or the constant value could be introduced as a parameter (to increase readability).
Otherwise this needs a comment.